release.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   branches: ['master'],
   plugins: [
     ['@semantic-release/commit-analyzer', { preset: 'conventionalcommits' }],
-    ['@semantic-release/npm', { pkgRoot: 'dist/' }],
```

Comment:
This bypasses the normal `package.json` file resolutions and ends up publishing a packages that is structurally different than the source code. This means that `yarn link` can't readily be used to test library changes with consuming projects.